### ISSUE TITLE
ImportError: No module named requests

### ISSUE THREAD

**Issue by bethesque** [pact-foundation/pact-python#31]
Is there a missing dependency declaration, or am I just a python noob?

```
$ pip install pact-python
Collecting pact-python
  Downloading pact-python-0.6.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/c8/ypphsm1d4gq3klcygh889hjm0000gn/T/pip-build-ffGYmj/pact-python/setup.py", line 14, in <module>
        import pact
      File "pact/__init__.py", line 2, in <module>
        from .consumer import Consumer
      File "pact/consumer.py", line 2, in <module>
        from .pact import Pact
      File "pact/pact.py", line 7, in <module>
        import requests
    ImportError: No module named requests
```

**Comment by hartror** [comment]
👋  @bethesque's token Python dev friend chiming in.

A side note on this issue. `tox -r` should have found this install bug but it appears that tox installs `requirements_dev.txt` which has `-e requirements.txt`. This of course means that the dependencies are installed before `setup.py` runs in tox.

**Comment by matthewbalvanz-wf** [comment]
This should be resolved in v0.6.1.
